recipe/recipe.yaml

```
@@ -0,0 +1,48 @@
+context:
+  name: sphinx-autosummary-accessors
+  version: 2025.3.1
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name | replace('-', '_') }}-${{ version }}.tar.gz
+  sha256: 41292dffe742703abe987f66c62e109d832490ff74d273d6512503e6066e456f
+
+build:
+  number: 0
```

shouldn't this bump the build number?
```suggestion
  number: 1
```